python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -715,6 +712,51 @@ def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList
         if nImage is not None:
             nImage.assign(subNImage, bbox)
 
+    def _removeMaskPlanes(self, maskedImage):
```

Comment:
In general, don't use a leading underscore in method names. It means "the user will never want to call this", which is very rarely the case.